### PR TITLE
feat(search): Implements search by job title

### PR DESCRIPTION
**Pull request by ayz4sci**
#### What does this PR do?
Implements job search by job title

#### How should this be manually tested?
- Pull the branch
- Run the following commands: 
`yarn`, 
`yarn relay`
`yarn start`

#### What are the relevant pivotal tracker stories?
- [#165274307](https://www.pivotaltracker.com/story/show/165274307)

### CODE REVIEW

**Review by CeciliaCaroline — COMMENTED** [review]
Looks good to me

File: src/components/topbar/topbar.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
-            <input type="text" name="search" placeholder="search for jobs " />
+            <input 
+              type="text" 
+              name="search" 
+              placeholder="search for job by title" 
+              onChange={(event) => {
+                handleSearch(event.target.value);
+              }}
+            />
             <Button type="submit" size="sm" color="primary">Search</Button>
```
Comment (CeciliaCaroline) [inline src/components/topbar/topbar.js]: What is the purpose of the search button? It does not seem to trigger any events

**Review by CeciliaCaroline — APPROVED** [review]

**Review by kisakyegordon — APPROVED** [review]
Looks good to me.

File: src/components/topbar/topbar.js (hunk 1 above)
Comment (kisakyegordon) [reply src/components/topbar/topbar.js]: +⒈

Comment (dotNesh) [reply src/components/topbar/topbar.js]: +1

**Review by dotNesh — COMMENTED** [review]
Looks good to me!

**Pull request merged**